requirements.txt

```diff
@@ -0,0 +1,5 @@
+fonttools==3.0
```

Comment:
The version on PyPI is old and is not the latest HEAD. I would recommend to pip install fonttools from the git repo instead